tests/e2e/specs/myOrders_spec.js

```diff
@@ -0,0 +1,33 @@
+describe('my orders', () => {
+  const customer = {
+    firstName: 'Charlie',
+    lastName: 'Bucket',
+    email: 'charlie.bucket+ci@commercetools.com',
+    password: 'p@ssword',
+  };
+
+  const cartDraft = {
+    customerEmail: 'charlie.bucket+ci@commercetools.com',
+    currency: 'EUR',
+    lineItems: {
+      sku: 'M0E20000000DPZ0',
+    },
+    shippingAddress: {
+      country: 'DE',
+    },
+  };
+
+  before(() => {
+    cy.login(customer);
+  });
+
+  it('shows my orders', () => {
+    cy.createMyOrder(cartDraft);
+    cy.get('[data-test=my-orders-button]').click();
+    cy.url().should('include', '/user/orders');
+    cy.get('[data-test=total-price]')
+      .should('exist');
+    cy.get('[data-test=total-price]')
+      .should('exist');
```

Comment:
You should not only check it exists like this, you should check value by value that it shows the correct information. Also, I would create at least a couple of orders and see that all of them appear (just checking that there are 2, not value by value for each one).

Reply:
This still applies. Do not check if a date exists, as this just checks that the `div` is displayed, but doesn't check anything about the content. It may perfectly be that no date is being printed. Use regular expressions or any other way to check that a date matches what you expect it to show.